HeterogeneousCore/SonicTriton/scripts/cmsTriton

```diff
@@ -315,14 +319,59 @@ wait_server(){
 	echo "server is ready!"
 }
 
+edit_model(){
+    cp -r $1 $TMPDIR/local_model_repo/
+    IFS='/' read -ra ADDR <<< "$1"
+    CONFIG=$TMPDIR/local_model_repo/${ADDR[-1]}/config.pbtxt
+
+    echo "" >> "$CONFIG"
+    echo "instance_group [" >> "$CONFIG"
+    echo "  {" >> "$CONFIG"
+    echo "    count: "$2 >> "$CONFIG"
+    echo "    kind: KIND_CPU" >> "$CONFIG"
+    echo "  }" >> "$CONFIG"
+    echo "]" >> "$CONFIG"
+    echo "" >> "$CONFIG"
+
+    if grep -Fq "onnx" "$CONFIG"
+    then
+	echo "parameters { key: \"intra_op_thread_count\" value: { string_value: \"1\" } }" >> "$CONFIG"
+	echo "parameters { key: \"inter_op_thread_count\" value: { string_value: \"1\" } }" >> "$CONFIG"
+    fi
+
+    if grep -Fq "tensorflow" "$CONFIG"
+    then
+	echo "parameters { key: \"TF_NUM_INTRA_THREADS\" value: { string_value: \"1\" } }" >> "$CONFIG"
+	echo "parameters { key: \"TF_NUM_INTER_THREADS\" value: { string_value: \"1\" } }" >> "$CONFIG"
+	echo "parameters { key: \"TF_USE_PER_SESSION_THREADS\" value: { string_value: \"1\" } }" >> "$CONFIG"
+    fi
+}
+
 list_models(){
 	# make list of model repositories
+        if [ "$INSTANCES" -gt 0 ]
+	then
+	    if [ -d "$TMPDIR/local_model_repo" ]
```

Comment:
put the name "local_model_repo" in a local variable somewhere to reduce duplication/hardcoding

Reply:
I put this in a global variable so that it can be used by both list_models and edit_model without being forwarded from one to the other.  Is that ok?  I can switch to a local variable if you'd prefer

Reply:
That's fine.